runtime/src/lib.rs

```diff
@@ -468,8 +346,25 @@ impl InstanceFilter<Call> for ProxyType {
 				Call::Balances(..)
 			),
             ProxyType::Governance => matches!(c,
-				Call::Democracy(..) | Call::Council(..) | Call::Elections(..)
-			)
+				// Call::Democracy(..) |
```

Comment:
Should we add this already? https://github.com/centrifuge/centrifuge-chain/pull/341/files

Reply:
can come in a different PR else this will get big